### PR TITLE
Buttons: More sentence-casing for button labels

### LINKED ISSUE #40120: Buttons: More sentence casing of labels
As a followup to #39692 I noticed a few stray Title Casing on button labels. Let's track those down and make them Sentence case.

**My Home:** `View Site`
Screen Shot 2020-03-13 at 9 41 45 AM

**Domain contact info:** `Save Contact Info`
Screen Shot 2020-03-13 at 9 43 01 AM

**Domain contact info:** `Edit Contact Info`
image

**Domain transfer to another user:** `Transfer Domain`
Screen Shot 2020-03-13 at 9 43 39 AM

**Domain transfer:** `Update Settings and Continue`
Screen Shot 2020-03-13 at 9 43 46 AM

**Cart dropdown on Domains and Plans:** `Add a Plan`
Screen Shot 2020-03-13 at 9 43 59 AM

**Earn section:** `Earn Free Credits`
Screen Shot 2020-03-13 at 9 46 32 AM

**People > Invites > Invite Details:** `Clear Invite`
Screen Shot 2020-03-13 at 9 47 29 AM

### PR DESCRIPTION
**Pull request by danhauk**
#### Changes proposed in this Pull Request

* Fixing a few stray uses of `Title Case` button labels to use `Sentence casing`

#### Testing instructions

* Checkout this branch locally or use https://calypso.live/?branch=update/more-button-sentence-casing
* Visit the pages listed below to check the buttons have proper casing.

**My Home**
Screen Shot 2020-03-16 at 11 20 47 AM

**Domains > { registered domain } > Contacts and Privacy**
image

**Domains > { registered domain } > Contacts and Privacy > Edit contact info**
image

**Domains > { registered domain } > Transfer Domain > Transfer to another registrar**
image

**Domains > { registered domain } > Transfer Domain > Transfer to another user**
image

**Domains (with empty cart) > Cart button in section header**
image

**People > Invites > Invite Details**
image

**Settings > Delete your site permanently**
image

**Settings > Delete your site permanently > click `Delete site`**
image

Fixes #40120

### CODE REVIEW

**Comment by danhauk** [comment]
@akirk Should I still add the `[Status] String Freeze` label here? Per your [previous comment](https://github.com/Automattic/wp-calypso/pull/39692#issuecomment-596500069) it may not be needed here, but I wanted to confirm.

File: client/components/domains/use-your-domain-step/index.jsx [hunk 1]
```diff
@@ -382,7 +382,7 @@ class UseYourDomainStep extends React.Component {
 			translate( "Requires changes to the domain's DNS" ),
 			this.getMappingPriceText(),
 		];
-		const buttonText = translate( 'Map Your Domain' );
+		const buttonText = translate( 'Map your domain' );
```
Comment (a8ci18n) [inline client/components/domains/use-your-domain-step/index.jsx]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

File: client/my-sites/domains/domain-management/contacts-privacy/index.jsx [hunk 2]
```diff
@@ -78,7 +78,7 @@ class ContactsPrivacy extends React.PureComponent {
 							this.props.selectedDomainName
 						) }
 					>
-						{ translate( 'Edit Contact Info' ) }
+						{ translate( 'Edit contact info' ) }
```
Comment (a8ci18n) [inline client/my-sites/domains/domain-management/contacts-privacy/index.jsx]: :warning: This change will be queued for retranslation. We'll display the English text until then.

File: client/my-sites/domains/domain-management/edit-contact-info/form-card.jsx [hunk 3]
```diff
@@ -168,7 +168,7 @@ class EditContactInfoFormCard extends React.Component {
 				</FormLabel>
 				<DesignatedAgentNotice
 					domainRegistrationAgreementUrl={ domainRegistrationAgreementUrl }
-					saveButtonLabel={ translate( 'Save Contact Info' ) }
+					saveButtonLabel={ translate( 'Save contact info' ) }
```
Comment (a8ci18n) [inline client/my-sites/domains/domain-management/edit-contact-info/form-card.jsx]: :warning: This change will be queued for retranslation. We'll display the English text until then.

File: client/my-sites/domains/domain-management/transfer/transfer-out/locked.jsx [hunk 4]
```diff
@@ -80,7 +80,7 @@ class Locked extends React.Component {
 						primary
 						disabled={ this.state.submitting }
 					>
-						{ translate( 'Update Settings And Continue' ) }
+						{ translate( 'Update settings and continue' ) }
```
Comment (a8ci18n) [inline client/my-sites/domains/domain-management/transfer/transfer-out/locked.jsx]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

File: client/my-sites/domains/domain-management/transfer/transfer-to-other-user/index.jsx [hunk 5]
```diff
@@ -181,7 +181,7 @@ class TransferOtherUser extends React.Component {
 				},
 				{
 					action: 'confirm',
-					label: this.props.translate( 'Confirm Transfer' ),
+					label: this.props.translate( 'Confirm transfer' ),
```
Comment (a8ci18n) [inline client/my-sites/domains/domain-management/transfer/transfer-to-other-user/index.jsx]: :warning: This change will be queued for retranslation. We'll display the English text until then.

File: client/my-sites/domains/domain-management/transfer/transfer-to-other-user/index.jsx [hunk 6]
```diff
@@ -215,7 +215,7 @@ class TransferOtherUser extends React.Component {
 		const { selectedDomainName: domainName, translate, users, selectedSite } = this.props,
 			availableUsers = this.filterAvailableUsers( users ),
 			{ currentUserCanManage, domainRegistrationAgreementUrl } = getSelectedDomain( this.props ),
-			saveButtonLabel = translate( 'Transfer Domain' );
+			saveButtonLabel = translate( 'Transfer domain' );
```
Comment (a8ci18n) [inline client/my-sites/domains/domain-management/transfer/transfer-to-other-user/index.jsx]: :warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **22** times:
`translate( 'Transfer a Domain' )` *ES Score: 7*

File: client/my-sites/earn/home.tsx [hunk 7]
```diff
@@ -195,7 +195,7 @@ const Home: FunctionComponent< ConnectedProps > = ( {
 		}
 
 		const cta: CtaButton = {
-			text: translate( 'Earn Free Credits' ) as string,
+			text: translate( 'Earn free credits' ) as string,
```
Comment (a8ci18n) [inline client/my-sites/earn/home.tsx]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

File: client/my-sites/people/people-invite-details/index.jsx [hunk 8]
```diff
@@ -85,7 +85,7 @@ export class PeopleInviteDetails extends React.PureComponent {
 					scary={ isPending }
 					onClick={ this.handleDelete }
 				>
-					{ isPending ? translate( 'Revoke Invite' ) : translate( 'Clear Invite' ) }
+					{ isPending ? translate( 'Revoke invite' ) : translate( 'Clear invite' ) }
```
Comment (a8ci18n) [inline client/my-sites/people/people-invite-details/index.jsx]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

Comment (a8ci18n) [inline client/my-sites/people/people-invite-details/index.jsx]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

File: client/my-sites/site-settings/delete-site/index.jsx [hunk 9]
```diff
@@ -140,16 +140,16 @@ class DeleteSite extends Component {
 		const deleteButtons = [
 			<Button onClick={ this.closeConfirmDialog }>{ translate( 'Cancel' ) }</Button>,
 			<Button primary scary disabled={ deleteDisabled } onClick={ this._deleteSite }>
-				{ translate( 'Delete this Site' ) }
+				{ translate( 'Delete this site' ) }
```
Comment (a8ci18n) [inline client/my-sites/site-settings/delete-site/index.jsx]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

File: client/my-sites/site-settings/delete-site/index.jsx [hunk 10]
```diff
@@ -140,16 +140,16 @@ class DeleteSite extends Component {
...
 			</Button>,
 		];
 
 		const strings = {
-			confirmDeleteSite: translate( 'Confirm Delete Site' ),
-			contactSupport: translate( 'Contact Support' ),
-			deleteSite: translate( 'Delete Site' ),
-			exportContent: translate( 'Export Content' ),
-			exportContentFirst: translate( 'Export Content First' ),
+			confirmDeleteSite: translate( 'Confirm delete site' ),
```
Comment (a8ci18n) [inline client/my-sites/site-settings/delete-site/index.jsx]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

File: client/my-sites/site-settings/delete-site/index.jsx [hunk 11]
```diff
@@ -140,16 +140,16 @@ class DeleteSite extends Component {
...
+			contactSupport: translate( 'Contact support' ),
+			deleteSite: translate( 'Delete site' ),
```
Comment (a8ci18n) [inline client/my-sites/site-settings/delete-site/index.jsx]: :warning: This change will be queued for retranslation. We'll display the English text until then.

File: client/my-sites/site-settings/delete-site/index.jsx [hunk 12]
```diff
@@ -140,16 +140,16 @@ class DeleteSite extends Component {
...
+			exportContent: translate( 'Export content' ),
```
Comment (a8ci18n) [inline client/my-sites/site-settings/delete-site/index.jsx]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

File: client/my-sites/site-settings/delete-site/index.jsx [hunk 13]
```diff
@@ -140,16 +140,16 @@ class DeleteSite extends Component {
...
+			exportContentFirst: translate( 'Export content first' ),
```
Comment (a8ci18n) [inline client/my-sites/site-settings/delete-site/index.jsx]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

**Comment by danhauk** [comment]
Failing tests appear unrelated to this PR.

**Comment by sixhours** [comment]
These look good!

I found more cases in the domains section, and on some of the empty placeholder screen states. Did you want to add these to this PR or should I open a new issue?

**Comment by danhauk** [comment]
@sixhours Thanks! Feel free to add them to this PR.

**Comment by sixhours** [comment]
**Domain-only site placeholder screen under Manage**

Screen Shot 2020-03-16 at 12 19 59 PM

Screen Shot 2020-03-16 at 12 19 43 PM

**Free site -> Email**

Screen Shot 2020-03-16 at 12 17 25 PM

**Name servers and DNS management screen**

Screen Shot 2020-03-16 at 12 17 01 PM

**Domain management**

Should these button-like-tab things be Sentence case, too?

Screen Shot 2020-03-16 at 12 16 00 PM

**WordPress.com site address**

Screen Shot 2020-03-16 at 12 14 23 PM

**Comment by jancavan** [comment]
Thanks for working on this, Dan.

I found a few on Reader:

**Reader empty state (when you're not following any sites):**

Screen Shot 2020-03-16 at 1 52 00 PM

**/following/manage empty state:**

Screen Shot 2020-03-16 at 1 51 44 PM

**/read/conversations:**

Screen Shot 2020-03-16 at 1 53 57 PM

**/activities/likes:**

Screen Shot 2020-03-16 at 1 53 52 PM

**Comment by akirk** [comment]
@danhauk should be ok without but please ping our team immediately after merge so that we can take the changed translations online. Thanks!

**Comment by sixhours** [comment]
New changes look good too 👍

**Comment by danhauk** [comment]
The one regarding Discover on Reader is an interesting one:

image

Since "Discover" is the feature/product name it makes sense for it to be capitalized. Which is why it's capitalized in the sentence above the button. But `Explore Discover` sounds strange if I don't read and understand that "Discover" is the name of the feature.

I left it alone for now and it's not a blocker for this PR, but wondering if you had any thoughts on a copy update for that label. Maybe something like `Discover sites` or `Explore sites` or even just `Explore`?

File: client/my-sites/domains/domain-management/name-servers/index.jsx [hunk 14]
```diff
@@ -299,7 +299,7 @@ class NameServers extends React.Component {
 				isPlaceholder={ this.isLoading() }
 				path={ domainManagementDns( this.props.selectedSite.slug, this.props.selectedDomainName ) }
 			>
-				{ this.props.translate( 'DNS Records' ) }
+				{ this.props.translate( 'DNS records' ) }
```
Comment (a8ci18n) [inline client/my-sites/domains/domain-management/name-servers/index.jsx]: :warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **26** times:
`translate( 'New DNS records saved.' )` *ES Score: 9*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/more-button-sentence-casing)

File: client/my-sites/domains/domain-management/edit/registered-domain.jsx [hunk 15]
```diff
@@ -137,7 +137,7 @@ class RegisteredDomain extends React.Component {
 
 		return (
 			<VerticalNavItem path={ path }>
-				{ this.props.translate( 'Name Servers and DNS' ) }
+				{ this.props.translate( 'Name servers and DNS' ) }
```
Comment (a8ci18n) [inline client/my-sites/domains/domain-management/edit/registered-domain.jsx]: :warning: This change will be queued for retranslation. We'll display the English text until then.

File: client/my-sites/domains/domain-management/edit/wpcom-domain.jsx [hunk 16]
```diff
@@ -114,8 +114,8 @@ class WpcomDomain extends React.Component {
 				}
 			>
 				{ isWpcomDomain
-					? this.props.translate( 'Change Site Address' )
-					: this.props.translate( 'Edit Site Address' ) }
+					? this.props.translate( 'Change site address' )
+					: this.props.translate( 'Edit site address' ) }
```
Comment (a8ci18n) [inline client/my-sites/domains/domain-management/edit/wpcom-domain.jsx]: :warning: This change will be queued for retranslation. We'll display the English text until then.

File: client/my-sites/domains/domain-management/list/domain-only.jsx [hunk 17]
```diff
@@ -56,11 +56,11 @@ const DomainOnly = ( { primaryDomain, hasNotice, recordTracks, siteId, slug, tra
 			<EmptyContent
 				title={ translate( '%(domainName)s is ready when you are.', { args: { domainName } } ) }
 				line={ translate( 'Start a site now to unlock everything WordPress.com can offer.' ) }
-				action={ translate( 'Create Site' ) }
+				action={ translate( 'Create site' ) }
```
Comment (a8ci18n) [inline client/my-sites/domains/domain-management/list/domain-only.jsx]: :warning: This change will be queued for retranslation. We'll display the English text until then.

File: client/my-sites/domains/domain-management/name-servers/custom-nameservers-form.jsx [hunk 18]
```diff
@@ -120,11 +120,11 @@ class CustomNameserversForm extends React.PureComponent {
 							onClick={ this.handleSubmit }
 							disabled={ this.props.submitDisabled }
 						>
-							{ translate( 'Save Custom Name Servers' ) }
+							{ translate( 'Save custom name servers' ) }
```
Comment (a8ci18n) [inline client/my-sites/domains/domain-management/name-servers/custom-nameservers-form.jsx]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

File: client/my-sites/domains/domain-management/name-servers/custom-nameservers-form.jsx [hunk 19]
```diff
@@ -120,11 +120,11 @@ class CustomNameserversForm extends React.PureComponent {
...
 						</FormButton>
 
 						<FormButton type="button" isPrimary={ false } onClick={ this.handleReset }>
-							{ translate( 'Reset to Defaults' ) }
+							{ translate( 'Reset to defaults' ) }
```
Comment (a8ci18n) [inline client/my-sites/domains/domain-management/name-servers/custom-nameservers-form.jsx]: :warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **35** times:
`translate( 'Settings reset to defaults.' )` *ES Score: 8*

File: client/my-sites/email/email-management/index.jsx [hunk 20]
```diff
@@ -172,7 +172,7 @@ class EmailManagement extends React.Component {
 		}
 
 		const emailForwardingAction = {
-			secondaryAction: translate( 'Add Email Forwarding' ),
+			secondaryAction: translate( 'Add email forwarding' ),
```
Comment (a8ci18n) [inline client/my-sites/email/email-management/index.jsx]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

File: client/reader/stream/empty.jsx [hunk 21]
```diff
@@ -34,7 +34,7 @@ class FollowingEmptyContent extends React.Component {
 					onClick={ this.recordAction }
 					href="/read/search"
 				>
-					{ this.props.translate( 'Find Sites to Follow' ) }
+					{ this.props.translate( 'Find sites to follow' ) }
```
Comment (a8ci18n) [inline client/reader/stream/empty.jsx]: :warning: This change will be queued for retranslation. We'll display the English text until then.

File: client/reader/tag-stream/empty.jsx [hunk 22]
```diff
@@ -41,7 +41,7 @@ class TagEmptyContent extends React.Component {
 				onClick={ this.recordAction }
 				href="/read"
 			>
-				{ this.props.translate( 'Back to Following' ) }
+				{ this.props.translate( 'Back to following' ) }
```
Comment (a8ci18n) [inline client/reader/tag-stream/empty.jsx]: :warning: This change will be queued for retranslation. We'll display the English text until then.

**Comment by jancavan** [comment]
@danhauk Yeah, that's true about Discover.

**Comment by sixhours** [comment]
"Explore Discover" is awkward wording and almost looks like a bug out of context.

> Maybe something like Discover sites or Explore sites or even just Explore?

I think just "Explore" works well.

File: client/reader/search-stream/empty.jsx [hunk 23]
```diff
@@ -41,7 +41,7 @@ class SearchEmptyContent extends React.Component {
 				onClick={ this.recordAction }
 				href="/read"
 			>
-				{ this.props.translate( 'Back to Following' ) }
+				{ this.props.translate( 'Back to following' ) }
```
Comment (a8ci18n) [inline client/reader/search-stream/empty.jsx]: :warning: This change will be queued for retranslation. We'll display the English text until then.

**Comment by danhauk** [comment]
I updated all instances of "Explore Discover" to just "Explore". This one is ready for review! cc @sixhours @jancavan @automattic/manage

**Comment by jancavan** [comment]
@danhauk I'm still seeing this one, Dan. But the rest LGTM.

Screen Shot 2020-03-18 at 8 38 00 AM

**Comment by sixhours** [comment]
LGTM 👍

File: client/blocks/conversations/empty.jsx [hunk 24]
```diff
@@ -34,7 +34,7 @@ class ConversationsEmptyContent extends React.Component {
 					onClick={ this.recordAction }
 					href="/read/search"
 				>
-					{ this.props.translate( 'Find Posts to Follow' ) }
+					{ this.props.translate( 'Find posts to follow' ) }
```
Comment (a8ci18n) [inline client/blocks/conversations/empty.jsx]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

File: client/reader/site-stream/empty.jsx [hunk 25]
```diff
@@ -45,7 +45,7 @@ const SiteEmptyContent = ( { translate } ) => {
 			onClick={ recordSecondaryAction }
 			href="/read/search"
 		>
-			{ translate( 'Find Sites to Follow' ) }
+			{ translate( 'Find sites to follow' ) }
```
Comment (a8ci18n) [inline client/reader/site-stream/empty.jsx]: :warning: This change will be queued for retranslation. We'll display the English text until then.

File: client/reader/liked-stream/empty.jsx [hunk 26]
```diff
@@ -36,7 +36,7 @@ class TagEmptyContent extends React.Component {
 					onClick={ this.recordAction }
 					href="/read"
 				>
-					{ this.props.translate( 'Back to Following' ) }
+					{ this.props.translate( 'Back to following' ) }
```
Comment (mmtr) [inline client/reader/liked-stream/empty.jsx]: Should we keep "Following" in title case? Seems like a feature/section name that lists followed blogs.

File: client/reader/list-stream/empty.jsx [hunk 27]
```diff
@@ -36,7 +36,7 @@ class ListEmptyContent extends React.Component {
 					onClick={ this.recordAction }
 					href="/read"
 				>
-					{ this.props.translate( 'Back to Following' ) }
+					{ this.props.translate( 'Back to following' ) }
```
Comment (mmtr) [inline client/reader/list-stream/empty.jsx]: Same here.

File: client/reader/tag-stream/empty.jsx (hunk 22 above)
Comment (mmtr) [inline client/reader/tag-stream/empty.jsx]: And one more.

File: client/reader/liked-stream/empty.jsx (hunk 26 above)
Comment (danhauk) [reply client/reader/liked-stream/empty.jsx]: Good catch here @mmtr. You're right, "Following" in this case in the feature/section name.

File: client/reader/liked-stream/empty.jsx [hunk 28]
```diff
@@ -45,13 +45,13 @@ class TagEmptyContent extends React.Component {
...
 				>
-					{ this.props.translate( 'Explore Discover' ) }
+					{ this.props.translate( 'Explore' ) }
 				</a>
 			) : null;
 
 		return (
 			<EmptyContent
-				title={ this.props.translate( 'No Likes Yet' ) }
+				title={ this.props.translate( 'No likes yet' ) }
```
Comment (a8ci18n) [inline client/reader/liked-stream/empty.jsx]: :ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.

File: client/reader/search-stream/empty.jsx [hunk 29]
```diff
@@ -63,7 +63,7 @@ class SearchEmptyContent extends React.Component {
 
 		return (
 			<EmptyContent
-				title={ this.props.translate( 'No Results' ) }
+				title={ this.props.translate( 'No results' ) }
```
Comment (a8ci18n) [inline client/reader/search-stream/empty.jsx]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **93** times:
`translate( 'No results.' )` *ES Score: 9*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/more-button-sentence-casing)

**Pull request merged**